### PR TITLE
Add lint and format commands on makefile.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
## Fixes / Features
- Add lint and format commands on makefile.

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Review by pawloch00 — DISMISSED** [review]

**Comment by mbobrovskyi** [comment]
cc: @pawloch00 @IrvingMg

File: .github/workflows/lint_and_format.yml [hunk 1]
```diff
@@ -33,17 +33,10 @@ jobs:
...
-        pylint $(git ls-files '*.py')
+      run: make pylint
     - name: Check format with pyink
-      run: pyink --check . 
-
+      run: make pyink
     - name: Run pytype
-      run: pytype --config=pytype-conf.cfg
\ No newline at end of file
+      run: make pytype
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply .github/workflows/lint_and_format.yml]: Done

File: Makefile [hunk 2]
```diff
@@ -14,40 +14,83 @@ PROJECT_DIR := $(realpath $(shell dirname $(firstword $(MAKEFILE_LIST))))
...
+pyink:
+	pyink --check .
+
+.PHONY: pyink-fix
+pyink-fix:
+	pyink .
+
+.PHONY: pytype
+pytype:
+	pytype --config=pytype-conf.cfg
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply Makefile]: Done

File: Makefile [hunk 3]
```diff
@@ -14,40 +14,83 @@ PROJECT_DIR := $(realpath $(shell dirname $(firstword $(MAKEFILE_LIST))))
...
-.PHONY: install install_kjob install_kueuectl install_gcloud check_python update-path 
-
+.PHONY: install
 install: check-python check-gcloud install-kueuectl install-kjob pip-install
 
-install-dev: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install install-pytest 
+.PHONY: install-dev
+install-dev: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install install-pytest
 
+.PHONY: pip-install
```
Comment (IrvingMg) [inline Makefile:23]: It looks like for targets we're following the pattern `install-[dep_name]`. Can we rename this to `install-pip`?

Comment (mbobrovskyi) [reply Makefile:23]: With pip-install we are installing dependencies but not a pip.

**Review by IrvingMg — APPROVED** [review]

**Comment by mbobrovskyi** [comment]
cc: @pawloch00

**Comment by pawloch00** [comment]
Could you please add a Readme section describing usage of make lint for checking if code is formatted properly?

**Review by pawloch00 — APPROVED** [review]

**Pull request merged**